main.c

```diff
@@ -26,38 +26,33 @@ static bool debug = false;
       fprintf(stderr, "DEBUG| " fmt "\n", __VA_ARGS__);                        \
   } while (0)
 
-static void print_vmnet_status(const char *func, vmnet_return_t v) {
+#define INFOF(fmt, ...)     fprintf(stderr, "INFO | " fmt "\n", __VA_ARGS__)
+#define ERROR(msg)          fprintf(stderr, "ERROR| " msg "\n")
+#define ERRORF(fmt, ...)    fprintf(stderr, "ERROR| " fmt "\n", __VA_ARGS__)
+#define ERRORN(name)        ERRORF(name ": %s", strerror(errno))
```

Comment:
This should move to log.h so we can split the to code to multiple files and use consistent logging.

Reply:
I guess we should use https://developer.apple.com/documentation/os/os_log ?

Reply:
See `man 3 os_log` for the usage

Reply:
Sure, I'll try this.

But I think we should keep the logging macros so it is easy to change the logging backend later.

Reply:
Do you plan to update this?

Reply:
What I have here was good enough for testing and fixing various issue, and I did not have time to check this direction so far.

I can update with with the version I run locally on top of current master, and we can add an issue to consider os_log.

We anyway need to add timestamps, using os_log will avoid this work.

Reply:
Added an issue to consider using os_log: https://github.com/lima-vm/socket_vmnet/issues/64